Mathlib/NumberTheory/ModularForms/EisensteinSeries/IsBoundedAtImInfty.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+
+import Mathlib.Analysis.Complex.UpperHalfPlane.FunctionsBoundedAtInfty
+import Mathlib.Analysis.Normed.Order.Lattice
+import Mathlib.NumberTheory.ModularForms.EisensteinSeries.UniformConvergence
+import Mathlib.NumberTheory.ModularForms.Identities
+
+/-!
+# Boundedness of Eisenstein series
+
+We show that Eisenstein series of weight `k` and level `Γ(N)` with congruence condition
+`a : Fin 2 → ZMod N` are bounded at infinity.
+
+## Outline of argument
+
+We need to bound the value of the Eisenstein series (acted on by `A : SL(2,ℤ)`)
+at a given point `z` in the upper half plane. Since these are modular forms of level `Γ(N)`,
+it suffices to prove this for `z ∈ verticalStrip N z.im`.
+
+We can then, first observe that the slash action just changes our `a` to `(a ᵥ* A)` and
+we then use our bounds for Eisenstein series in these vertical strips to get the result.
+-/
+
+noncomputable section
+
+open ModularForm EisensteinSeries UpperHalfPlane Set Filter Function Complex Matrix
+  SlashInvariantForm
+
+open scoped Topology BigOperators Nat Classical MatrixGroups
```

Comment:
What happens if you don't open `Classical`?

Reply:
nothing, so I've removed it!